### ISSUE TITLE
next_in_days not reached 0

### ISSUE THREAD

**Issue by FlexxFR** [xirixiz/homeassistant-afvalwijzer#151]
### Issue

Yesterday, a waste pick-up day, the value next_in_days did not reach 0 (and that is why my automation didn't alert me and almost forgot my container....automations make you lazy ;) )

Anyway, in my HA history I see the value did not reach 0 like it normally does.
This time it went from 1 directly to 7.
![image](https://user-images.githubusercontent.com/65705235/141610066-2caf191f-e7e8-4d42-8705-1bf389540232.png)

The value of sensor.afvalwijzer_today was OK during that time and set to the proper waste type. To values in afvalwijzer were up-to-date. 
![image](https://user-images.githubusercontent.com/65705235/141610153-2db09210-af97-4e1d-8495-c110abd7c9e2.png)

### Which version has the issue?

2021.11.01

### Anything in the logs that might be useful?

```txt
Looking at the these value changes in history, when is afvalwijzer updated?
Sometimes at 00:45, 10:45?
```

**Comment by xirixiz** [comment]
Funny thing is that a bugfix has caused this issue :)

I fixed this issue and therfor this issue is now introduced. Let me check it and create a new release this weekend.
https://github.com/xirixiz/homeassistant-afvalwijzer/issues/149

**Comment by xirixiz** [comment]
What happened before.

If the waste type and the first next waste type are equal (f.e. PMD Today and PMD Tomorrow), then PMD Tomorrow is skipped as for each trash type already a date is in the list.

What I did to fix it is to only look at trashtypes without taking Today into account, and therefor the 0 value in your case skipped, as it's only looking at the next waste type after Today.

I didn't have this issue as I always send out a notification on the evening before pickup (value 1 in hass).

**Comment by xirixiz** [comment]
Released 2021.11.02.

**Comment by FlexxFR** [comment]
Thanks for the fast bug fix!
